### PR TITLE
Added inactive state class to save button

### PR DESCRIPTION
**Pull request by codegaze**
This adds a 'btn-inactive' class to all save buttons on inactive state. I believe the inactive state isn't really straightforward to the user right now, and having an inactive class that can be used anytime is a plus imo :)

Changed the mixin to accept more parameters with default values like  $cursor: pointer, $opacity: 1, $hover: true in order to avoid losing consistency. 

Hope you like it :smiley_cat:

### CODE REVIEW

File: src/styles/main.scss [hunk 1]
```diff
@@ -77,6 +77,10 @@ i {
   @include btn($btn-active, $btn-active-border);
 }
 
+.btn-inactive {
+  @include btn($btn-inactive, $btn-inactive-border, default, .4, false);
```
Comment (mertkahyaoglu) [inline src/styles/main.scss]: What about `.5` for opacity? It's hard to see the text on the button. At least for my computer 😬

**Comment by mertkahyaoglu** [comment]
Thanks @codegaze I liked the idea of separating inactive state 👍

**Comment by mertkahyaoglu** [comment]
Can we apply the same style for the button in Configuration view?

**Comment by codegaze** [comment]
Missed that!
Morning + :coffee: + The Class and we are ready :wink:

**Comment by codegaze** [comment]
Now we are ready!
@mertkahyaoglu Do you think the save button for save/edit could be abstracted to its own component with safety?

**Comment by mertkahyaoglu** [comment]
Just one comment and ready to merge!

> Do you think the save button for save/edit could be abstracted to its own component?

Yeah, this PR would be a lot easy if we had a `SaveButton` component as we would change only one file. I suggest creating a new PR for that. Maybe a generic component for all types of buttons, Save, View, Create..

File: src/styles/main.scss (hunk 1 above)
Comment (codegaze) [inline src/styles/main.scss]: Updated. My monitor home shows colors by proximity. :joy:

**Comment by mertkahyaoglu** [comment]
Thanks @codegaze 🎉

**Pull request merged**